### PR TITLE
Alterwaycasestudy

### PR DESCRIPTION
**Pull request by JohnMarksRH**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

- Reformatted title so that it matches the title format in most of the other use cases. The title is aligned with the text beneath it. It my view this looks better than the current version, with the title indented to the left, and with a subtitle below.

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): @jmarks

This pull request needs review by: @bproffitt @mykaul

### CODE REVIEW

File: source/blog/2017-02-27-oVirt-gamification.html.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ You can find more info and details on the GetBadges game on the their blog site.
 * [Understanding Badges](https://getbadges.io/blog/understanding-badges-in-our-it-gamification-platform)
 * [How to motivate developers](http://getbadges.io/blog/how-to-motivate-developers)
 
-Game on!
+Game on!
```
Comment (mykaul) [inline source/blog/2017-02-27-oVirt-gamification.html.md]: I'm not sure what this change is about - whitespace damage?

Comment (duck-rh) [inline source/blog/2017-02-27-oVirt-gamification.html.md]: same remark

File: source/blog/2017-02-27-oVirt-gamification.html.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 title: oVirt Gamification--The oVirt Game You Didn't Know you Were Playing
 author: eedri
+date: 2017-02-27 13:05:00 UTC
```
Comment (duck-rh) [inline source/blog/2017-02-27-oVirt-gamification.html.md]: unrelated change. why changing TZ anyway?

File: source/blog/2017-02-27-oVirt-gamification.html.md [hunk 3]
```diff
@@ -47,4 +47,4 @@ You can find more info and details on the GetBadges game on the their blog site.
 * [Understanding Badges](https://getbadges.io/blog/understanding-badges-in-our-it-gamification-platform)
 * [How to motivate developers](http://getbadges.io/blog/how-to-motivate-developers)
 
-Game on!
```
Comment (duck-rh) [inline source/blog/2017-02-27-oVirt-gamification.html.md]: ???
